assets/src/components/map.tsx

```diff
@@ -237,15 +244,11 @@ const Shape = ({ shape }: { shape: Shape }) => {
           center={[stop.lat, stop.lon]}
           radius={3}
         >
-          <Popup className="m-vehicle-map__stop-tooltip">
-            {stop.name}
-            {inTestGroup(MAP_BETA_GROUP_NAME) && (
-              <StreetViewButton
-                latitude={stop.lat}
-                longitude={stop.lon}
-              ></StreetViewButton>
-            )}
-          </Popup>
+          {inTestGroup(MAP_BETA_GROUP_NAME) ? (
+            <StopCard stop={stop} direction={direction} />
+          ) : (
+            <Popup className="m-vehicle-map__stop-tooltip">{stop.name}</Popup>
+          )}
         </CircleMarker>
```

Comment:
Good question - probably not. I'll check with design.

Reply:
So after discussion with design we've settled on using the same hover tooltip that's part of the designs for [the ticket to add stations to the map](https://app.asana.com/0/1200180014510248/1203442408293386/f) that you're working on. How would you like to coordinate on implementing that? I assume it will be a simple component that just takes a single prop for the text to display.

Reply:
Hmmm yeah there is one other prop to offset the tooltip based on the marker size -- How does this sound: 
assuming this PR will get merged before the station icons, if you make the change in this PR to only have the full connections tooltip on the mapPage, then as part of the stations ticket I can restyle those stop tooltips to match the station tooltips?

Reply:
Sounds good to me, thanks!